src/components/pager/__internal__/pager-navigation.spec.js

```diff
@@ -83,15 +82,15 @@ describe("Pager Navigation", () => {
           showPageSizeSelection: true,
           pageSizeSelectionOptions,
           currentPage: 1,
-          pageCount: 1,
+          pageCount: 3,
           totalRecords: 1,
           onLast,
         },
         mount
       );
       const navLinks = wrapper.find(StyledPagerLinkStyles);
-      const last = navLinks.last();
-      last.simulate("click");
+      const first = navLinks.first();
```

Comment:
Not sure this should have changed to first, as the `onLast` function will only be called when the `last` link is clicked

Reply:
Ah missed updating the spy to `onFirst`, the `onLast` callback will not work as it will be enabled here.